### PR TITLE
New login page based on Tailwind

### PR DESCRIPTION
**Pull request by Riscky**
I recently became a big fan of Tailwind because it gives me the control over the layout without ever leaving the html.

Tailwind needs webpack, so this PR is based off #775 

I copied our current color and breakpoint configuration to Tailwind to ease working with both Yeti and Tailwind

I merged the example login page into this branch, as it is as good as finished

### CODE REVIEW

File: app/views/layouts/doorkeeper.html.haml
Comment (Riscky) [inline app/views/layouts/doorkeeper.html.haml:2]: this changes are only for testing purposes and should be removed before merge

File: package.json
Comment (Riscky) [inline package.json:16]: I'm not actually sure what each of this packages do, so I'll have to clean this bit out

File: config/webpack/development.js
Comment (Riscky) [inline config/webpack/development.js:3]: this file is currently not run automatically on server start, which is annoying

File: config/webpacker.yml
Comment (Riscky) [inline config/webpacker.yml:1]: auto generated, but probably alright?

File: tailwind.config.js
Comment (Riscky) [inline tailwind.config.js:38]: I might want to remove this breakpoint, since it is not very efficient to generate css for a breakpoint like this XD

File: postcss.config.js
Comment (Riscky) [inline postcss.config.js:5]: same here, there might be steps here that don't make sense for us

File: tailwind.config.js
Comment (Mstiekema) [inline tailwind.config.js:10]: These*

File: tailwind.config.js
Comment (Mstiekema) [inline tailwind.config.js:9]: Isn't this a perfect spot to add the board color?

File: tailwind.config.js
Comment (Mstiekema) [inline tailwind.config.js:35]: These*

**Comment by Mstiekema** [comment]
Looks neat btw! Did however had to run `bundle exec rails webpacker:install` for some reason. Not sure if I did something wrong if it's actually required :/

File: tailwind.config.js
Comment (Riscky) [reply tailwind.config.js:9]: I guess it is, but then we could also not include a whole lot of the other colors and use a consistent color scheme instead. That feels a bit beyond the scope of this PR though.

**Comment by Riscky** [comment]
> Looks neat btw! Did however had to run `bundle exec rails webpacker:install` for some reason. Not sure if I did something wrong if it's actually required :/

This should defs not be necessary, but I will wait for #776 to be merged before diving into this one. #776 might actually fix build-related problems

**Review by Mstiekema — APPROVED** [review]
LGTM! :shipit:

**Pull request merged**
